doc/classes/Dictionary.xml

```diff
@@ -11,17 +11,26 @@
 		Creating a dictionary:
 		[codeblocks]
 		[gdscript]
-		var my_dir = {} # Creates an empty dictionary.
-		var points_dir = {"White": 50, "Yellow": 75, "Orange": 100}
-		var another_dir = {
-		    key1: value1,
-		    key2: value2,
-		    key3: value3,
+		var my_dict = {} # Creates an empty dictionary.
+		var dict_variable_key = "Another key name"
+		var dict_variable_value = "value2"
+		var points_dict = {"White": 50, "Yellow": 75, "Orange": 100}
+		var another_dict = {
+		    "Some key name": "value1",
+		    dict_variable_key: dict_variable_value,
+		}
```

Comment:
Can we swap them around 

		var another_dict = {
		    "Some key name": "value1",
		    dict_variable_key: dict_variable_value,
		}
                var points_dict = {"White": 50, "Yellow": 75, "Orange": 100}

That points_dict has nothing to do with  previous variables and users may be confused with what is going on. I would even separate all concepts with a empty line so the concepts that work together are separate from unrelated code. 

Maybe even like this:

		var my_dict = {} # Creates an empty dictionary.

		var dict_variable_key = "Another key name"
		var dict_variable_value = "value2"
		var another_dict = {
		    "Some key name": "value1",
		    dict_variable_key: dict_variable_value,
		}

		var points_dict = {"White": 50, "Yellow": 75, "Orange": 100}